gen/definitions/ftd_nat_policy.yaml

```diff
@@ -0,0 +1,333 @@
+    ordered_list: true
+    attributes:
+      - model_name: id
+        type: String
+        description: Identifier of the manual nat rule.
+        resource_id: true
+        exclude_example: true
+        exclude_test: true
+      - model_name: description
+        type: String
+        description: Manual nat rule description
+        example: My manual nat rule 1
+      - model_name: enabled
+        type: Bool
+        description: Indicates if the rule is enabled.
+        example: true
+      - model_name: section
+        type: String
+        description: Name of section to which the rule belongs.
+        mandatory: true
+        data_path: [metadata]
+        enum_values:
+          - BEFORE_AUTO
+          - AFTER_AUTO
+        example: BEFORE_AUTO
+      - model_name: natType
+        tf_name: nat_type
+        type: String
+        description: Type of the rule
+        enum_values:
+          - STATIC
+          - DYNAMIC
+        example: STATIC
+        mandatory: true
+      - model_name: fallThrough
+        tf_name: fall_through
+        type: Bool
+        description: Fallthrough to Interface PAT (Destination Interface)
+        example: false
+      - model_name: interfaceInOriginalDestination
+        tf_name: interface_in_original_destination
+        type: Bool
+        description: Use interface address as original destination
+        example: false
+        exclude_test: true
+        exclude_example: true
+      - model_name: interfaceInTranslatedSource
+        tf_name: interface_in_translated_source
+        type: Bool
+        description: Translate source network to destination interface address
+        example: false
+        exclude_test: true
+        exclude_example: true
+      - model_name: interfaceIpv6
+        tf_name: ipv6
+        type: Bool
+        description: Use the IPv6 address of the destination interface for interface PAT.
+        example: false
+        exclude_test: true
+        exclude_example: true
+      - model_name: netToNet
+        tf_name: net_to_net
+        type: Bool
+        description: Net to Net Mapping
+        example: false
+        exclude_example: true
+        exclude_test: true
+      - model_name: noProxyArp
+        tf_name: no_proxy_arp
+        type: Bool
+        description: Do not proxy ARP on Destination Interface
+        example: false
+        exclude_example: true
+        exclude_test: true
+      - model_name: unidirectional
+        type: Bool
+        description: Whether the rule is unidirectional
+        example: true
+        exclude_test: true
+        exclude_example: true
+      - model_name: id
+        tf_name: source_interface_id
+        type: String
+        data_path: [sourceInterface]
+        description: ID of source interface
+        example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+        exclude_test: true
+        exclude_example: true
+      - model_name: id
+        tf_name: original_source_id
```

Comment:
Would it make sense to name this `original_source_network_id` to be more specific?

Reply:
Updated description, as per our discussion.